apps/password-reset/src/app/services/password-reset-config.service.ts

```diff
@@ -14,15 +14,36 @@ export class PasswordResetConfigService {
     private appConfigService: AppConfigService,
     private storeService: StoreService,
     private location: Location,
-    private authzSevice: AuthzResolverService,
-    private guiAuthResolver: GuiAuthResolver
+    private authzSevice: AuthzResolverService
   ) { }
 
   loadConfigs(): Promise<void> {
     return this.appConfigService.loadAppDefaultConfig()
       .then(() => this.appConfigService.loadAppInstanceConfig())
       .then(() => this.setApiUrl())
-      .then(() => this.appConfigService.setInstanceFavicon());
+      .then(() => this.appConfigService.setInstanceFavicon())
+      .then(() => {
+        const queryParams = location.search.substr(1);
+        if(!queryParams.includes('token')) {
+          return this.initAuthService.verifyAuth();
```

Comment:
the authentication is not working at all, I think the problem is in the thing that you dont wait for the result, there is not a promise in this {}

Reply:
and rebase please